core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -331,7 +331,7 @@ class RemoteServicesMockServer(port:Int) extends RawlsTestUtils {
       response()
         .withHeaders(jsonHeader)
         .withBody {
-          (1 to numWorkflows).map(_ => ExecutionServiceStatus(UUID.randomUUID().toString, "Submitted")).toList.toJson.toString
+          (1 to numWorkflows).map(n => ExecutionServiceStatus(UUID.randomUUID().toString, if (n == numWorkflows) "Failed" else "Submitted")).toList.toJson.toString
```

Comment:
Long story. :) 

This used to be hard coded to return 2 successes and 1 failure. In https://github.com/broadinstitute/rawls/pull/718 I changed it to be dynamic, but lost the failure. The tests still passed.

However when working on the instrumentation tests I found the test whose _intent_ it was to check for a failure -- the assertion was just weak (it was checking `status != Queued` or something). But while verifying instrumentation it actually failed due to lack of a failure. So I changed this logic to make the last workflow a failure in Cromwell.